test/unittests/t_dns.cc

```diff
@@ -953,6 +953,15 @@ TEST_F(T_Dns, HostfileResolverMultipleAddresses) {
   EXPECT_EQ(host.ipv6_addresses(), expected_ipv6);
 }
 
+TEST_F(T_Dns, HostfileResolverBlankLines) {
+  CreateHostfile("   \n  #comment\n127.0.0.1 localhost\n");
```

Comment:
Let's add more: an ip address without a host name, a valid line with a comment at the end, an empty line, a too long ip address / host name